scripts/gitlab/build_and_test.sh

```diff
@@ -106,7 +106,9 @@ then
     cmake \
       -C ${hostconfig_path} \
       ${project_dir}
-    cmake --build . -j
+    if ! cmake --build . -j; then
+      cmake --build . --verbose -j 1
```

Comment:
Looks good!

For clarity and being able to more easily search around in the log file, I would suggest:
```suggestion
      echo ""
      echo ""
      echo "Build failed, running build again sequentially in verbose mode"
      echo ""
      echo ""
      cmake --build . --verbose -j 1
```

Reply:
Done!